components/Layout/Footer.jsx

```diff
@@ -141,25 +141,27 @@ const LinksBlock = ({ title, links, className }) => (
     {links.map((item) => {
       const LinkTag = item.isExternal ? 'a' : Link;
       return (
-        <LinkTag
-          href={item.link}
-          key={item.title}
-          className="hover:text-black"
-          {...(item.isExternal
-            ? {
-              target: '_blank',
-              rel: 'noopener noreferrer',
-            }
-            : {})}
-        >
+        <span key={item.title} className="contents">
```

Comment:
this is just fixing not aligned footer, noticed that and couldn't pass by....
<img width="578" alt="image" src="https://github.com/valory-xyz/olas-website/assets/22725775/e07ba0f2-bc42-4d79-b6fa-a5b5f94f4724">